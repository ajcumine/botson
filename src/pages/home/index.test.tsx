import { shallow } from 'enzyme';
import React from 'react';
import Home from './';

describe('Home page', () => {
  it('renders', () => {
    const component = shallow(<Home />);
    expect(component).toMatchSnapshot();
  });
});
