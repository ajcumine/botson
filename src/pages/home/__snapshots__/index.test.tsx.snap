// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders 1`] = `
<p>
  To get started, edit 
  <code>
    src/App.tsx
  </code>
   and save to reload.
</p>
`;
